portfolio.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel = "stylesheet" href = "css/portfolio.css"/>
```

Comment:
nitpick: make sure that spacing before and after the equals signs in html attributes are consistent across files